.github/workflows/release-ninja-binaries.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         mkdir build && cd build
         cmake -DCMAKE_BUILD_TYPE=Release ..
         cmake --build . --parallel --config Release --target ninja
+        ctest -vv
```

Comment:
Will this print the output if a test fails?

Reply:
It should, yes.